src/components/about/about.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './about.scss'
+
+export default props=>{
+  return (
+    <div>
+      <h1 className="center">About Page</h1>
+    </div>
+  )
+}
```

Comment:
Not a huge deal but you should have an empty line at the end of all files, it keeps git happy.